OmniKitUI/ViewControllers/OmnipodSettingsViewController.swift

```diff
@@ -712,27 +710,15 @@ class OmnipodSettingsViewController: RileyLinkSettingsViewController {
         }
 
         confirmationBeepsTableViewCell.isLoading = true
-        if confirmationBeeps {
-            pumpManager.setConfirmationBeeps(enabled: false, completion: { (error) in
-                if let error = error {
-                    DispatchQueue.main.async {
-                        let title = LocalizedString("Error disabling confirmation beeps", comment: "The alert title for disable confirmation beeps error")
-                        self.present(UIAlertController(with: error, title: title), animated: true)
-                    }
+        pumpManager.setConfirmationBeeps(enabled: !pumpManager.confirmationBeeps, completion: { (error) in
+            if let error = error {
+                DispatchQueue.main.async {
+                    let title = LocalizedString("Error setting confirmation beeps", comment: "The alert title for setting confirmation beeps error")
+                    self.present(UIAlertController(with: error, title: title), animated: true)
                 }
-                done()
-            })
-        } else {
-            pumpManager.setConfirmationBeeps(enabled: true, completion: { (error) in
-                if let error = error {
-                    DispatchQueue.main.async {
-                        let title = LocalizedString("Error enabling confirmation beeps", comment: "The alert title for enable confirmation beeps error")
```

Comment:
Would prefer to keep the error message that the error happened while "enabling" or "disabling", vs just "setting".

Reply:
Agreed. Now using the original separate error strings for enabling and disabling confirmation beeps cases. This has the additional benefit of no additional internationalization work created by this PR.